pkg/cmd/deploy/deploy.go

```diff
@@ -116,8 +116,11 @@ func (cmd *DeployCmd) Run(f *cmdutil.Factory) error {
 	}
 
 	if Sync {
+		if len(ProjectConf) > 0 {
```

Comment:
You dont need to check for Len, because ProjectConf always has a value (if not sent, we use the default value "azion"

Reply:
I even thought about it but I ended up doing it anyway to make sure I can remove it here!